crates/rpc/src/server/mod.rs

```diff
@@ -57,8 +58,6 @@ impl Rpc {
             .build_v1alpha()
             .context("failed to build reflection service")?;
 
-        info!(target: COMPONENT, endpoint=?self.listener, store=%self.store, block_producer=?self.block_producer, "Server initialized");
```

Comment:
Should this or something similar be brought back? Feels like it would be interesting from a debugging perspective to know when it got to load the genesis, etc.

Reply:
Sure, i can rollback this change to re-add it.